modules/ref_fb_module/include/ref_fb_module/statistics_fb_impl.h

```diff
@@ -69,6 +112,11 @@ class StatisticsFbImpl final : public FunctionBlock
         }
     };
 
+    bool triggerMode;
+    FunctionBlockPtr nestedTriggerFunctionBlock;
+    InputPortPtr triggerOutput;
```

Comment:
`triggerInput` seems like a more fitting name.

Reply:
`triggerInput` instead of `triggerOutput`? But it is the output of the trigger here... It is however the input of the statistics function block at the moment.

Reply:
It's the input port into which a signal containing on/off trigger values is connected. I understand the reasoning, to me `triggerInput` feels more natural, but both are fine.